sequencer/src/main/java/net/consensys/linea/sequencer/liveness/LivenessPlugin.java

```diff
@@ -0,0 +1,398 @@
+      Function functionDown =
+          new Function(
+              "updateStatus",
+              Arrays.asList(new Bool(false), new Uint256(lastBlockTimestamp)),
+              Collections.emptyList());
+
+      String encodedFunctionDown = FunctionEncoder.encode(functionDown);
+      byte[] callDataBytesDown = Numeric.hexStringToByteArray(encodedFunctionDown);
+      Bytes callDataDown = Bytes.wrap(callDataBytesDown);
+
+      // Create first transaction
+      Transaction transactionDown = createTransaction(callDataDown);
+      submitTransactionViaJsonRpc(transactionDown);
+
+      // Second call: updateStatus(true, currentTimestamp)
+      Function functionUp =
+          new Function(
+              "updateStatus",
+              Arrays.asList(new Bool(true), new Uint256(currentTimestamp)),
+              Collections.emptyList());
+
+      String encodedFunctionUp = FunctionEncoder.encode(functionUp);
+      byte[] callDataBytesUp = Numeric.hexStringToByteArray(encodedFunctionUp);
+      Bytes callDataUp = Bytes.wrap(callDataBytesUp);
+
+      // Create second transaction
+      Transaction transactionUp = createTransaction(callDataUp);
+      submitTransactionViaJsonRpc(transactionUp);
+
+      log.info("Sequencer uptime transactions submitted via JSON-RPC.");
+    } catch (Exception e) {
+      log.error("Error sending sequencer uptime transaction", e);
+    }
+  }
+
+  private Transaction createTransaction(Bytes callData) throws IOException {
+    // Get nonce from the account
+    BigInteger nonce = BigInteger.ZERO;
+    if (web3j != null && credentials != null) {
+      EthGetTransactionCount ethGetTransactionCount =
+          web3j
+              .ethGetTransactionCount(
+                  credentials.getAddress(),
+                  org.web3j.protocol.core.DefaultBlockParameterName.LATEST)
+              .send();
+      if (ethGetTransactionCount.hasError()) {
+        throw new IOException(
+            "Error getting nonce: " + ethGetTransactionCount.getError().getMessage());
+      }
+      nonce = ethGetTransactionCount.getTransactionCount();
+    }
+
+    // TODO: get current gas price (currently using a 1 Gwei default value)
+    Wei gasPrice = Wei.of(1_000_000_000L);
+
+    // Create transaction
+    Transaction.Builder builder =
+        Transaction.builder()
+            .nonce(nonce.longValue())
+            .gasPrice(gasPrice)
+            .gasLimit(gasLimit)
+            .to(contractAddress)
+            .value(Wei.ZERO)
+            .payload(callData)
+            .chainId(chainId);
+
+    // Sign the transaction if credentials are available
+    if (credentials != null) {
+      // Create a raw transaction for signing
+      RawTransaction rawTransaction =
+          RawTransaction.createTransaction(
+              nonce,
+              gasPrice.getAsBigInteger(),
+              BigInteger.valueOf(gasLimit),
+              contractAddress.toString(),
+              BigInteger.ZERO,
```

Comment:
I would consider making this a constant for readability - `ZERO_TRANSACTION_VALUE` or similar?